#! /bin/sh /usr/share/dpatch/dpatch-run
## 31_debian_conf_doc.dpatch
## by Petter Reinholdtsen <pere@hungry.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Adjust the insserv configuration to match the debian init.d
## DP: script names.

@DPATCH@
diff -urNad insserv~/insserv.conf insserv/insserv.conf
--- insserv~/insserv.conf	2008-01-04 11:19:42.000000000 +0100
+++ insserv/insserv.conf	2008-01-04 11:21:59.000000000 +0100
@@ -1,29 +1,29 @@
 #
 # All local filesystems are mounted (done during boot phase)
 #
-$local_fs	boot.localfs
+$local_fs	+mountall +umountfs
 
 #
 # Low level networking (ethernet card)
 #
-$network	network +pcmcia +hotplug
+$network	+networking +ifupdown
 
 #
 # Named is operational
 #
-$named		+named +dnsmasq +lwresd $network
+$named		+named +dnsmasq +lwresd +bind9 $network
 
 #
 # All remote filesystems are mounted (note in some cases /usr may
 # be remote. Most applications that care will probably require
 # both $local_fs and $remote_fs)
 #
-$remote_fs	$local_fs +nfs
+$remote_fs	$local_fs +mountnfs +mountnfs-bootclean +umountnfs +sendsigs
 
 #
 # System logger is operational
 #
-$syslog		syslog
+$syslog		+syslog +sysklogd
 
 #
 # SunRPC portmapper available
@@ -33,9 +33,9 @@
 #
 # The system time has been set correctly
 #
-$time		boot.clock +xntpd
+$time		hwclock
 
 #
 # Services which need to be interactive
 #
-<interactive>   boot.crypto boot.localfs boot.rootfsck apache apache2 kdump
+<interactive>   udev mountdevsubfs checkroot checkfs console-screen
